src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -163,6 +163,7 @@ def load_arguments(self, _):
     for scope in ['disk create', 'snapshot create']:
         with self.argument_context(scope) as c:
             c.argument('source', help='source to create the disk/snapshot from, including unmanaged blob uri, managed disk id or name, or snapshot id or name')
+            c.argument('secure_vm_disk_encryption_set', help='Name or ID of disk encryption set created with ConfidentialVmEncryptedWithCustomerKey encryption type. Mandatory and required only when --security-type is set to ConfidentialVM_DiskEncryptedWithCustomerKey.')
```

Comment:
> Mandatory and required only when --security-type is set to ConfidentialVM_DiskEncryptedWithCustomerKey.

Please add relevant verification for this point



```suggestion
            c.argument('secure_vm_disk_encryption_set', help='Name or ID of disk encryption set created with ConfidentialVmEncryptedWithCustomerKey encryption type.')
```